src/dashboard/index.html

```diff
@@ -38,24 +38,42 @@ <h3>Bridge</h3>
 
     <section>
         <h3>Emulator</h3>
-        <div class="explain-note">Use left and right arrow keys to emulate the Trezor One buttons. Mouse clicks for Trezor T.</div><br>
-        <div id="emu-status-line"><b>Status: <span id="emu-status">unknown</span></b></div>
+        <div class="explain-note">
+            Use left and right arrow keys to emulate the Trezor One buttons. Mouse clicks for Trezor T.
+        </div>
+
+        <br>
+
+        <div id="emu-status-line">
+            <b>Status: <span id="emu-status">unknown</span></b>
+        </div>
         <div>
             <h4>Trezor One</h4>
             <select id="t1-select"></select>
             <button onclick="emulatorStart('t1-select');">Start</button>
-            <br />
         </div>
         <div>
             <h4>Trezor T</h4>
             <select id="t2-select"></select>
             <button onclick="emulatorStart('t2-select');">Start</button>
         </div>
-        <input id="wipeDevice" type="checkbox">
-        <label for="wipeDevice">Start with wiped/empty device</label><br>
-        <input id="emulatorUseLogfile" type="checkbox">
-        <label for="emulatorUseLogfile">Logs into logfile</label><br>
-        <hr />
+        <div>
+            <input id="wipeDevice" type="checkbox">
+            <label for="wipeDevice">Start with wiped/empty device</label>
+            <input id="emulatorUseLogfile" type="checkbox">
+            <label for="emulatorUseLogfile">Logs into logfile</label><br>
+        </div>
+        <div>
+            <!-- TODO: could be prefilling the t1-or-t2-select based on some keywords in URL
+            (like "trezor-emu-core" fills T2, "trezor.elf" fills T1)  -->
```

Comment:
Let's remove the comment, I think the select box is enough.

Reply:
Removed in the force-pushed commit (force-pushed twice, second already rebased on current master, even though there was no conflict)